libs/sdk-ui-dashboard/src/presentation/filterBar/utils.ts

```diff
@@ -2,7 +2,17 @@
 import { IntlShape } from "react-intl";
 import { messages } from "../../locales.js";
 import { IFilterButtonCustomIcon, VisibilityMode } from "@gooddata/sdk-ui-filters";
-import { DashboardDateFilterConfigModeValues } from "@gooddata/sdk-model";
+import {
```

Comment:
Please help to re-arrange the imports to make it more clear:
```javascript
import { IntlShape } from "react-intl";
import { IFilterButtonCustomIcon, VisibilityMode } from "@gooddata/sdk-ui-filters";
import {
    DashboardAttributeFilterConfigMode,
    DashboardAttributeFilterConfigModeValues,
    DashboardDateFilterConfigMode,
    DashboardDateFilterConfigModeValues,
} from "@gooddata/sdk-model";

import { messages } from "../../locales.js";
import {
    FilterBarAttributeItem,
    FilterBarAttributeItems,
    isFilterBarAttributeFilterPlaceholder,
} from "./filterBar/useFiltersWithAddedPlaceholder.js";
```

Reply:
updated